homepage.php

```diff
@@ -35,15 +32,41 @@
 				</ul>
 			</nav>
 		</div>
-		<div class="content_1"><!-- Holds the Welcome and introduction about the page -->
+		<div class="img_1">
+			<img src = "images/img.png">
+		</div>
+		<div class="content_1"><!-- Holds the main page content -->
 			<h1>Welcome to MUSIC MP3</h1>
 			<p>MUSIC MP3 is a free music player application. With a high-quality music store, you will have a great music experience.</p>
 			<p>Digital music service with millions of high quality copyrighted songs , helping you listen to music, download music, upload and sync my music store on multiple devices.</p>
 		</div>
-		<div class="content_2"><!-- Holds the About Us information -->
```

Comment:
In the layout 2 I think it should have one div box to introduce to use about the music database so I remove the other div box.